snippets/findify-product-card.liquid

```diff
@@ -29,7 +29,7 @@
         height="{{ card_product.featured_media.height }}"
       />
     </div>
-    <div>{{ card_product.title }}</div>
+    <div class='findify-swiper-product-title'>{{ card_product.title }}</div>
```

Comment:
This snippet is in common between autocomplete, search and recommendation.

Put a findify-product-title class

Then within your findify-recommendation use something in this lines:

`.recommendation-wrapper .findify-product-title {}`

Reply:
ok)